### ISSUE TITLE
Cannot click Tour link in About/FAQ page

### ISSUE THREAD

**Issue by tobsecret** [jupyterlab/jupyterlab#2705]
When opening the Jupyterlab FAQ, there seems to be no way of clicking the Jupyterlab tour.
Below is the HTML that I see when inspecting the following line:

>Check out the JupyterLab tour here

**Comment by jzf2101** [comment]
@tobsecret Let me take a look and see what I can find

**Comment by tobsecret** [comment]
The following is the HTML that I get in the browser:
```HTML
<li class="jp-FAQ-answer">Check out the JupyterLab tour <a class="jp-FAQ-a" data-commandlinker-command="about-jupyterlab:open" data-commandlinker-args="null">here</a></li>
```

**Comment by blink1073** [comment]
We removed the tour because it was out of date, that link should be removed as well.

**Comment by blink1073** [comment]
Fixed by #2785 (there is no more tour link).
